content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,132 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <path>] [-x X] [-y Y]
+                        [--no-csv-header] [--title <text>] [--x-label <text>]
+                        [--y-label <text>] [--unset [<prop> [<prop> ...]]]
+                        target
+
+positional arguments:
+  target                Plot file to set props to.
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_display properties_ (such as `y-label`, `template`, `title`, etc.) each time
+plots are generated with `dvc plot show` or `dvc plot diff`. This command sets
+(or unsets) default display props for specific plot files.
+
+The available display properties are expressed as the [options](#options) of
+this command (prefixed with `--`).
+
+> Display props are based on the
+> [Vega specification](https://vega.github.io/vega/docs/specification/)
+
+The `target` metrics file should be an <abbd>output</abbr> of one of the
+[DVC pipeline](/doc/command-reference/pipeline) stages (see the `--plots` option
+of `dvc run`), listed in a
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories) file. `dvc plots modify`
+adds the options to `dvc.yaml`.
+
+## Options
+
+- `-t <path>, --template <path>` - set a default
+  [plot template](/doc/command-reference/plots#plot-templates).
+
+- `-x <field>` - set a default field name from which the X axis data comes from.
+
+- `-y <field>` - set a default field name from which the Y axis data comes from.
+
+- `--x-label <text>` - set a default title for the X axis.
+
+- `--y-label <text>` - set a default title for the Y axis.
+
+- `--title <text>` - set a default plot title.
+
+- `--unset [<prop> [<prop> ...]]` - unset one or more display properties. Use
+  the prop name(s) without `--` in the argument sent to this option.
+
+- `--no-csv-header` - lets DVC know that CSV or TSV `targets` do not have a
+  header.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+The initial plot was showing the last column of CSV file by default which is
+_loss_ metrics while _accuracy_ is expected as Y axis:
+
+```csv
```

Comment:
since we don't have it now, let's remove? I mean `csv` spec

Reply:
Removed from all 4 cmd refs in 5a92508b.